scripts/abins/spowdersemiempiricalcalculator.py

```diff
@@ -576,10 +606,36 @@ def _calculate_s_over_threshold(s=None, freq=None, coeff=None):
 
         return freq, coeff
 
+    def _calculate_order_one_dw(self, *, q2: np.ndarray,
+                                frequencies: np.ndarray,
+                                a_tensor=None, a_trace=None,
+                                b_tensor=None, b_trace=None,
+                                indices=None):
+        """
+        Calculate mode-dependent Debye-Waller factor for the first order quantum event for one atom.
+        :param q2: squared values of momentum transfer vectors
+        :param frequencies: frequencies for which transitions occur
+        :param indices: (unused) array which stores information how transitions can be decomposed in terms of fundamentals
```

Comment:
will this be used in the future?

Reply:
Probably not, it can come out 😄 

(It's an artefact of when this function would be passed around and needed a consistent interface with higher orders.)